src/dtclient/kubernetes_settings_test.go

```diff
@@ -201,9 +204,29 @@ func TestDynatraceClient_GetSettingsForMonitoredEntities(t *testing.T) {
 }
 
 func TestDynatraceClient_CreateOrUpdateKubernetesSetting(t *testing.T) {
+	t.Run(`create settings with monitoring for the given monitored entity id`, func(t *testing.T) {
+		// arrange
+		dynatraceServer := httptest.NewServer(mockDynatraceServerSettingsHandler(1, testObjectID, http.StatusOK))
+		defer dynatraceServer.Close()
+
+		skipCert := SkipCertificateValidation(true)
+		dtc, err := NewClient(dynatraceServer.URL, apiToken, paasToken, skipCert)
+		require.NoError(t, err)
+		require.NotNil(t, dtc)
+
+		// act
+		actual, err := dtc.(*dynatraceClient).CreateOrUpdateKubernetesSetting(testName, testUID, testScope)
+
+		// assert
+		assert.NotNil(t, actual)
```

Comment:
use `require` to quick fail tests and not pollute output with follow up errors (see above, makes most sense for `NotNil`/`NoError` checks)

Reply:
thanks for input, done